src/pages/Lending/Pool/components/Repay/RepayInput.tsx

```diff
@@ -338,6 +341,7 @@ export const RepayInput = ({
   }, [_isSmartContractAddress])
 
   const quoteErrorTranslation = useMemo(() => {
+    if (!isThorchainLendingRepayEnabled) return 'trade.errors.tradingNotActiveNoAssetSymbol'
```

Comment:
Might as well make a new translation for this one. Feels weird showing trades temp halted on repayment.

suggestion: `Repayments are temporarily halted.`

Reply:
b8c43b0ad4 - added some lending-specific copies for borrowing too while at it 

<img width="484" alt="image" src="https://github.com/shapeshift/web/assets/17035424/61038702-4363-4163-88cd-60db595523db">
<img width="474" alt="image" src="https://github.com/shapeshift/web/assets/17035424/0b0d7fab-d193-45f6-a9bf-d75def2abebb">